packages/liferay-npm-scripts/src/config/babel.json

```diff
@@ -3,7 +3,13 @@
 	"plugins": [
 		"@babel/proposal-class-properties",
 		"@babel/proposal-export-namespace-from",
-		"@babel/proposal-object-rest-spread"
+		"@babel/proposal-object-rest-spread",
+		[
+			"@babel/transform-modules-commonjs",
+			{
+				"strictMode": false
```

Comment:
In case anybody is wondering where this option is documented it's in https://babeljs.io/docs/en/babel-plugin-transform-strict-mode/ and not in https://babeljs.io/docs/en/next/babel-plugin-transform-modules-commonjs.html as you might expect.